src/app/dashboard/dashboard.component.css

```diff
@@ -18,6 +18,32 @@
   font-size: 0;
 }
 
+#homeHeader > .bs-container {
+  display: flex;
+  align-items: center;
+}
+
+#homeHeader > .bs-container > .navigation {
+  display: flex;
+  gap: 20px;
+  margin-left: 20px;
+}
+
+#homeHeader > .bs-container > .navigation > a {
+  font-size: 14px;
+  color: var(--kg-gray);
+  text-decoration: none;

```

Comment:
Can we remove these customizations and use the default link color? (That makes the contrast a lot better too.) `text-decoration` is already `none` by default for this application's links.

```suggestion
```